src/datatypes/src/schema/column_schema.rs

```diff
@@ -61,7 +61,7 @@ pub struct ColumnSchema {
     pub data_type: ConcreteDataType,
     is_nullable: bool,
     is_time_index: bool,
-    default_constraint: Option<ColumnDefaultConstraint>,
+    pub default_constraint: Option<ColumnDefaultConstraint>,
```

Comment:
Don't publish this field and use `with_default_constraint()`. It can do some validation.